### PR TITLE
User-Defined APIs - Updated troubleshooting page

### PR DESCRIPTION
**Pull request by VanRobaysWouter**

### CODE REVIEW

File: user-guide/Advanced_Modules/User-Defined_APIs/Troubleshooting/UD_APIs_Troubleshooting.md [hunk 1]
```diff
@@ -50,13 +50,14 @@ The errors in this section can be generated when the UserDefinableApiEndpoint Dx
...
+  DataMiner should install ASP.NET Core 6 automatically, but in case this failed or in case it has been removed, the installer can generate this error. Since .NET 8, DataMiner doesn't install .NET anymore.
 
   To fix this:
 
-  1. Go to the [.NET 6 download page](https://dotnet.microsoft.com/en-us/download/dotnet/6.0) and download the latest ASP.NET Core Runtime 6.
+  1. Go to the .NET download page of the version matching your DxM version. and download the latest ASP.NET Core Runtime Hosting Bundle.
+      - You can find the matching version on the [UserDefinableApiEndpoint page](xref:UD_APIs_UserDefinableApiEndpoint#version).
 
-  1. On the DMA, got to the folder `C:\Skyline DataMiner\Tools\ModuleInstallers\` and run the installer `DataMiner UserDefinableApiEndpoint 1.X.X.X.msi`.
+  1. On the DMA, go to the folder `C:\Skyline DataMiner\Tools\ModuleInstallers\` and run the installer `DataMiner UserDefinableApiEndpoint 1.X.X.X.msi`.
```
Comment (Skyline-ThomasGH) [inline user-guide/Advanced_Modules/User-Defined_APIs/Troubleshooting/UD_APIs_Troubleshooting.md:60]:
```suggestion
  1. On the DMA, go to the folder `C:\Skyline DataMiner\Tools\ModuleInstallers\` and run the installer `DataMiner UserDefinableApiEndpoint X.X.X.X.msi`.
```

File: user-guide/Advanced_Modules/User-Defined_APIs/Troubleshooting/UD_APIs_Troubleshooting.md [hunk 2]
```diff
@@ -72,6 +73,10 @@ The errors in this section can be generated when the UserDefinableApiEndpoint Dx
 
   When the DxM is upgraded or automatically repaired during the execution of a DataMiner upgrade package, it can occur that the files in question are locked. This prevents the installer from continuing until they are unlocked. The installer will log a message if this is the case, and it will continuously check every 5 seconds for a maximum of 60 seconds until the files are unlocked. If they are still locked, this message is logged, and the installer will try to continue. This may result in files not being copied, and other errors may occur. The log lines mentioned should contain the names of the process or processes that are locking the files. You may need to terminate these and try the upgrade again.
 
+- **Serivce 'DataMiner UserDefinableApiEndpoint' failed to start. Verify that you have sufficient privileges to start system services.**
```
Comment (Skyline-ThomasGH) [inline user-guide/Advanced_Modules/User-Defined_APIs/Troubleshooting/UD_APIs_Troubleshooting.md:76]:
```suggestion
- **Service 'DataMiner UserDefinableApiEndpoint' failed to start. Verify that you have sufficient privileges to start system services.**
```

File: user-guide/Advanced_Modules/User-Defined_APIs/UD_APIs_UserDefinableApiEndpoint.md [hunk 3]
```diff
@@ -39,7 +39,10 @@ Below you can find a list of all the *UserDefinableApiEndpoint* DxM versions and
 | 3.2.0       | .NET 6                | 10.4.0+                            | 10.4.3/10.4.0                    |
 
 > [!NOTE]
-> Versions that are not listed in the overview above were not released in official DataMiner upgrade packages.
+>
+> - Versions that are not listed in the overview above were not released in official DataMiner upgrade packages.
+> - Make sure you always download the ASP.NET Core Runtime Hosting bundle.
```
Comment (Skyline-ThomasGH) [inline user-guide/Advanced_Modules/User-Defined_APIs/UD_APIs_UserDefinableApiEndpoint.md:44]: Maybe word it that it has to be installed? This now may sound incomplete.

**Review by Skyline-ThomasGH — APPROVED** [review]

**Review by Skyline-ThomasGH — APPROVED** [review]

**Comment by EdithLansens** [comment]
@VanRobaysWouter, I've made some changes. Is this still OK content-wise? If so, the pull request is ready to merge.

**Review by EdithLansens — APPROVED** [review]

**Pull request merged**